### ISSUE TITLE
Install / Deployment Guide

### ISSUE THREAD

**Issue by prohealthbox** [Microsoft/HealthClinic.biz#16]
Hi,

Can anyone please tell me / guide me to deploy this complete solution.
can i just open 10_Demos_Deployment and deploy to Azure ..?

Sorry for posting this in Issues Section, Mods kindly move this to approperiate section.

**Comment by ibonilm** [comment]
Hi,

All the existing docs is in the wiki https://github.com/Microsoft/HealthClinic.biz/wiki 

Please, check this section.

**Comment by ibonilm** [comment]
The 10_Demos_Deployment  solution contains an ARM template the creates the Azure services (AppService, Mobile Service, SQL Server) needed to deploy the backend services used by the client mobile apps.

**Comment by prohealthbox** [comment]
@ibonilm 
Yes , I have tried that and was sucessfull in creating Azure Services (AppService, Mobile Service, SQL Server)
After that what is the next setp..?

**Comment by erikaehrli** [comment]
Hello,

Please see the Azure Developer Quickstarts section. You will find guides by topic.

Also the Wiki has docs for each project.

Thanks,
Erika

**Comment by prohealthbox** [comment]
hi
i have successfully deployed webapp and mobileapp, but  i have an issue in local system as well as in azure,
when i login to private area in portal with admin or user credentials , there is no data, below is the screenshot. Please help me, Database connection everything is fine.
![22](https://cloud.githubusercontent.com/assets/21254969/18137753/5b01d2c0-6fc7-11e6-9e78-7cc2ba5d716f.png)

**Comment by ibonilm** [comment]
Hi,

Check that the node dependencies are well installed and the gulp default task runs when you build the project...or run it manually.

**Comment by prohealthbox** [comment]
hi, all dependencies are installed,  i found the error related to lib.js , screenshot below ,please helpme
![3](https://cloud.githubusercontent.com/assets/21254969/18193510/ee3e24aa-70fb-11e6-815b-e4e8f0a429bb.png)

**Comment by ibonilm** [comment]
I would remove the node_modules and run "npm install" and "gulp" command to be sure that the dependencies are well installed and the default gulp task works fine.

**Comment by prohealthbox** [comment]
hi , i have removed node_modules and ran "npm install" and "gulp" command,but have issues with successfully running code, can anyone please help me to fix this issue.i can share my desktop via remote. my mail id is arjun.kumar21@gmail.com , Please help me to resolve this.
